lib/api/collection_config.rb

```diff
@@ -0,0 +1,120 @@
+module Api
+  class CollectionConfig
+    def self.names_for_feature(product_feature_name)
+      names_for_features[product_feature_name]
+    end
+
+    def self.name_for_klass(resource_klass)
```

Comment:
@abellotti @carbonin if we can settle on the design, I'll write some specs for this method (and others)